diff --git a/dist/glob.js b/dist/glob.js
index 0c1feb4286d95a2b195baa65d68a5697974e70f5..c795db8f7c2f4ab3935c86371a2a48775216e614 100644
--- a/dist/glob.js
+++ b/dist/glob.js
@@ -4,9 +4,7 @@ import * as nativeFs from "fs";
 import { readdir, readdirSync, realpath, realpathSync, stat, statSync } from "fs";
 import { basename, dirname, isAbsolute, normalize, posix, relative, resolve, sep } from "path";
 import { fileURLToPath } from "url";
-import { createRequire } from "module";
 //#region ../../node_modules/.pnpm/fdir@6.5.0_picomatch@4.0.5/node_modules/fdir/dist/index.mjs
-var __require = /* @__PURE__ */ createRequire(import.meta.url);
 function cleanPath(path) {
 	let normalized = normalize(path);
 	if (normalized.length > 1 && normalized[normalized.length - 1] === sep) normalized = normalized.substring(0, normalized.length - 1);
@@ -384,12 +382,7 @@ var APIBuilder = class {
 		return sync(this.root, this.options);
 	}
 };
-let pm = null;
-/* c8 ignore next 6 */
-try {
-	__require.resolve("picomatch");
-	pm = __require("picomatch");
-} catch {}
+let pm = require_picomatch();
 var Builder = class {
 	globCache = {};
 	options = {
